calico-enterprise/operations/logstorage/create-storage.mdx

```diff
@@ -98,7 +98,7 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: tigera-elasticsearch
-provisioner: kubernetes.io/aws-ebs
+provisioner: ebs.csi.aws.com
```

Comment:
the EBS parameters are listed [here](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/parameters.md#supported-parameters)

They would have to be changed as follows, I believe:

```yaml
parameters:
  type: gp2
  "csi.storage.k8s.io/fstype": ext4
```